### PR TITLE
Update bazel workspace and version to latest.

### PR DESCRIPTION
**Pull request by jverkoey**
This should resolve the kokoro build breakages we've been witnessing recently.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -12,12 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-git_repository(
+http_archive(
```
Comment (yarneo) [inline WORKSPACE:15]: I'm a fan of using http_archive over git_repository for speed and security, but we should be mindful when updating versions that it's more than just a tag update, but rather checking archive name, sha'ing, and updating the prefix.

File: .kokoro [hunk 2]
```diff
@@ -74,6 +74,12 @@ run_bazel() {
 
   fix_bazel_imports
 
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    bazel version
+    use_bazel.sh 0.11.0
```
Comment (yarneo) [inline .kokoro]: Do we prefer this over `use_bazel.sh latest`?

**Review by yarneo — APPROVED** [review]
LGTM % few comments

File: .kokoro (hunk 2 above)
Comment (jverkoey) [reply .kokoro]: Ah, good catch. `use_bazel.sh latest` is probably better.

File: WORKSPACE (hunk 1 above)
Comment (jverkoey) [reply WORKSPACE:15]: Agreed that the overhead is a bit annoying. [Internal](http://cl/173579337) and [external](https://docs.bazel.build/versions/master/be/workspace.html#git_repository) guidance implies that we may want/need to move over to http_archive unfortunately.

Comment (yarneo) [reply WORKSPACE:15]: makes sense!

**Pull request merged**